data/companies.yml

```diff
@@ -221,9 +225,9 @@ integrations:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
 
 pro_services:
-  - name: "Your Name Here"
-    logo: "/logos/istio.svg"
-    url: "https://istio.io/"
+  - name: "Solo.io"
+    logo: "/logos/solo.svg"
+    url: "https://solo.io/"
     description: "Submit a PR to add your company here!"
```

Comment:
This section is for people who provide consulting and support for upstream Istio, as opposed to the support you provide with Gloo Mesh Enterprise f.ex - that's accounted for in the providers tab.

If you do provide this kind of support, we can talk about listing you twice; please add an (optionally) multi-line description here, explaining what services you provide.

Reply:
@craigbox  Solo does provide services including training and support for upstream Istio customers independent of whether they are using Gloo Mesh. See more about our free public [training and certification](https://www.solo.io/blog/get-certified-on-the-essentials-for-istio/) and [enterprise production support](https://www.solo.io/blog/enterprise-support-for-istio-in-production/).

Reply:
ack, please make that clear in the PR